### PR TITLE
Model/add.unit tests of converts

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**
ok adding a unit tests. please review.

### CODE REVIEW

File: src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java [hunk 1]
```diff
@@ -28,32 +28,30 @@
...
-        CriteriaPPTtoImage criterion = (CriteriaPPTtoImage) criteriaConvert;
+        CriteriaPptxToImage criteria = (CriteriaPptxToImage) criteriaConvert;
         try {
-            File file = new File(criterion.getSrcPath());
+            File file = new File(criteria.getSrcPath());
             XMLSlideShow ppt = new XMLSlideShow(new FileInputStream(file));
             Dimension pgsize = ppt.getPageSize();
             List<XSLFSlide> slide = ppt.getSlides();
             BufferedImage img = null;
             for (int i = 0; i < slide.size(); i++) {
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java]: Please the variable change for one more explicit the variable i to the for.

File: src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java [hunk 2]
```diff
@@ -28,32 +28,30 @@
...
                 img = new BufferedImage(pgsize.width, pgsize.height, BufferedImage.TYPE_INT_RGB);
                 Graphics2D graphics = img.createGraphics();
-
                 graphics.setPaint(Color.white);
                 graphics.fill(new Rectangle2D.Float(0, 0, pgsize.width, pgsize.height));
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java]: That is "(0, 0, pgsize.width, pgsize.height)" 0 look magical numbers.

**Review by LimbertVargas — COMMENTED** [review]
it is all ok for me, good Job Josue

File: src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java]: variable?

Comment (TheYisusGithub) [reply src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java]: the variable "i" to the for I think you cant use for example index.

Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java]: please modify your variable name "i" to a more descriptive name.

File: src/main/java/com/fundation/webservice/model/CriteriaPptxToImage.java [hunk 3]
```diff
@@ -15,11 +15,11 @@
  * @author Josue Rodriguez
  * @version 1.0
  */
-public class CriteriaPPTtoImage extends CriteriaConvert {
+public class CriteriaPptxToImage extends CriteriaConvert {
     private String ext;
 
 	/**
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/webservice/model/CriteriaPptxToImage.java]: Please align the comments /**.

File: src/main/java/com/fundation/webservice/model/CriteriaPptxToImage.java [hunk 4]
```diff
@@ -30,7 +30,7 @@ public String getExt() {
...
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/webservice/model/CriteriaPptxToImage.java:30]: Align the comments. /**

File: src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java]: fix complete.

File: src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java]: fix, complete

File: src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java (hunk 1 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java]: fix complete.

File: src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java [hunk 5]
```diff
@@ -28,38 +28,38 @@
...
-            for (int i = 0; i < slide.size(); i++) {
+            for (int index = 0; index < slide.size(); index++) {
                 img = new BufferedImage(pgsize.width, pgsize.height, BufferedImage.TYPE_INT_RGB);
                 Graphics2D graphics = img.createGraphics();
-
                 graphics.setPaint(Color.white);
-                graphics.fill(new Rectangle2D.Float(0, 0, pgsize.width, pgsize.height));
-
+                graphics.fill(new Rectangle2D.Float(POS_X, POS_Y, pgsize.width, pgsize.height));
                 //render
```
Comment (TheYisusGithub) [inline src/main/java/com/fundation/webservice/model/ConvertPptxToImage.java:54]: Please remove this comment I think is not nesserary.

**Pull request merged**